tests/test_change_debt.py

```diff
@@ -31,8 +35,19 @@ def test_change_debt(
     vault.updateStrategyDebtRatio(strategy, currentDebt / 2, {"from": gov})
     # sleep for a day to make sure we are swapping enough (Uni v3 combined with only 6 decimals)
     chain.sleep(86400)
+    chain.mine(1)
+
+    # Validate that the gauge is rewarding CRV and GNO
+    assert gauge.claimable_tokens.call(strategy, {"from": strategy}) > 0, "No CRV"
```

Comment:
note that with the introduction of ySwaps, strategies don't make money in a fork, unless the developer somehow mocks the swaps for LP tokens. We didn't mock the swaps but we did include tests to validate that the gauge has claimable rewards, and that we are able to claim them